transformer_lens/components.py

```diff
@@ -1318,6 +1321,23 @@ def forward(
         )
 
 
+def add_head_dimension(
```

Comment:
This feels weird as a standalone function in a file full of classes. Can you make it a method on TransformerBlock? (Since it's not used anywhere else).

And the name feels a bit misleading to me, when I read it I think of adding a dummy axis, a la `x -> x[:, None, :]`. Maybe "expand along head dimension" or "repeat along head dimension"?

Reply:
I agree its a little awkward but its also used by `BertBlock`. Should I just put it in utils?

Reply:
Fixed